### DISCUSSION TITLE
Users and Groups

### DISCUSSION THREAD

**Discussion by isaiahgoveait** [sct/overseerr#2160] [Feature Requests]
Hello,

There should be a new feature where we can create custom groups. And select certain users to add to those groups.

There should also be options and features you can add or remove from each group, i.e. giving a "premium membership group" radarr movie availability: announced

And

Giving another group "standard membership group" radarr movie availability: released

Or 

A certain group to have the ability to not see available media.

This is to allow flexible control within one autonomous server environment.

**Comment by samwiseg0** [comment]
Hopefully your “friends” are not paying for access to your servers as that would violate the Plex TOS. 

For that reason I don’t see this feature being implemented in this form to support/condone those actions.

**Reply by isaiahgoveait** [reply]
Yup just friends, Sam please stop assuming there are payments involved. This feature would introduce a much better organizational system.

**Reply by isaiahgoveait** [reply]
You're telling me you don't have premium friends? haha

**Reply by samwiseg0** [reply]
> Yup just friends, Sam please stop assuming there are payments involved. This feature would introduce a much better organizational system.

There is no assumption. Read your post. Standard membership vs premium membership. That is an odd way to organize your so called friends. 

No I don’t have premium friends.
